roofit/roofitcore/src/RooBinWidthFunction.cxx

```diff
@@ -29,11 +29,25 @@
 #include "RooDataHist.h"
 #include "RunContext.h"
 
+bool RooBinWidthFunction::_enabled = true;
+
+void RooBinWidthFunction::enable() {
+  _enabled = true;
+}
+
+bool RooBinWidthFunction::enabled() {
+  return _enabled;
+}
+
+void RooBinWidthFunction::disable() {
+  _enabled = false;
+}
```

Comment:
Can you please add a doxygen line for each of these?
Like:
```C++
/// Does this and that.
```

Reply:
sure!